Update index.mdx (#1053)
hashicorp/terraform-plugin-sdk: website/docs/plugin/sdkv2/index.mdx

---
page_title: 'Home - Plugin Development: SDKv2'

> description: The Terraform Plugin SDK is a legacy Go module for developing Terraform providers.

Author: ```suggestion
description: Maintain plugins built on the legacy SDK.
```
